### PR TITLE
VEN-386 & VEN-476 | Company customers

### PR DESCRIPTION
**Pull request by igordavydsson**
## Description :sparkles:

- Add `Company` model
- Expose company info through GQL API
- Update `Meta` options for `CustomerProfile` and `Boat` models
- Make the fields of Profile and BerthProfile node in GQL consistent
- Fix missing ProfileNodes' resolution logic

## Issues :bug:

Closes [VEN-381](https://helsinkisolutionoffice.atlassian.net/browse/VEN-381), [VEN-476](https://helsinkisolutionoffice.atlassian.net/browse/VEN-476)

## Testing :alembic:
**Automated tests ⚙️**

```shell
$ pytest customers/
```

**Manual testing 👷** 

```graphql
query GetBerthProfiles {
  berthProfiles {
    edges {
      node {
        id
        invoicingType
        comment
        company {
          businessId
          name
        }
        boats {
          id
        }
        berthApplications {
          edges {
            node {
              id
            }
          }
        }
        berthLeases {
          edges {
            node {
              id
            }
          }
        }
      }
    }
  }
}
```

## Screenshots :camera_flash:

image

### CODE REVIEW

File: customers/models.py [hunk 1]
```diff
@@ -21,6 +21,11 @@ class CustomerProfile(TimeStampedModel, UUIDModel):
     )
     comment = models.TextField(verbose_name=_("comment"), blank=True)
 
+    class Meta:
+        verbose_name = _("customer profile")
+        verbose_name_plural = _("customer profiles")
+        ordering = ("id",)
```
Comment (a-rmz) [inline customers/models.py]: Does it make sense ordering by `id` if it's a UUID?

File: customers/tests/test_customers_schema.py [hunk 2]
```diff
@@ -11,10 +22,152 @@ def test_profile_node_gets_extended_properly():
...
+
+    berth_application = BerthApplicationFactory(customer=customer_profile)
+    berth_lease = BerthLeaseFactory(customer=customer_profile)
+    company = CompanyFactory(customer=customer_profile)
+    boat = BoatFactory(owner=customer_profile)
+
+    query = QUERY_BERTH_PROFILE % berth_profile_id
+
+    # only superusers and profile owners can see profile node info
+    gql_user = superuser if is_superuser else customer_profile.user
```
Comment (a-rmz) [inline customers/tests/test_customers_schema.py]: Nice trick 👍

File: customers/models.py (hunk 1 above)
Comment (igordavydsson) [reply customers/models.py]: It has absolutely no effect 😅 `"id"` is default field for ordering anyway, just wanted to make it explicit.

Comment (igordavydsson) [reply customers/models.py]: no other field could fit better for odering at this point

File: customers/tests/test_customers_schema.py (hunk 2 above)
Comment (igordavydsson) [reply customers/tests/test_customers_schema.py]: tanks ❤️

File: customers/models.py (hunk 1 above)
Comment (a-rmz) [reply customers/models.py]: Alright :) just curious if there was any other expected result. 👍

**Pull request merged**